src/common/progress_bar.cpp

```diff
@@ -25,11 +25,16 @@ int ProgressBar::GetCurrentPercentage() {
 
 void ProgressBar::Start() {
 #ifndef DUCKDB_NO_THREADS
```

Comment:
Maybe the progress bar changes could be a separate PR?

Reply:
Yeah, I got carried away, should have done this in a different PR, I think right now is a bit annoying to remove it, but I'll keep in mind to have shorter/smaller PRs in the future. Sorry for that.